setup.py

```diff
@@ -50,8 +50,8 @@
             'numpy>=1.7',
             'pandas>=0.13.1',
             'datacache>=0.4.2',
-	    'pyfaidx',
-	    'memoized-property',
+            'pyfaidx>=0.3.4',
+            'memoized-property>=1.0.2',
```

Comment:
Instead of duplicating the requirements, can we parse requirements.txt in setup.py? 

Example: http://stackoverflow.com/a/16624700/481326